workflow/scripts/score.R

```diff
@@ -0,0 +1,87 @@
+#SCRIPT FOR COMPUTING ANCIENT_METAGENOME SCORE PER MICROBE
+#RUN SCRIPT AS: Rscript score.R RMA6_FILE MALTEXTRACT_OUTPUT NAME_LIST OUPUT_DIR
+
+args<-commandArgs(trailingOnly=TRUE)
+RMA6<-basename(args[1])
+MaltExtract_output<-args[2]
+name_list<-args[3]
+output_dir<-args[4]
```

Comment:
Isn't the output_dir simply the dirname of the name_list file? Is the argument to keep it as a parameter that we may want to change the output directory and therefore be able to set it via the CLI?

Reply:
Right now, the output_dir coincides with the name_list, indeed, but in the future it will not. I wanted to have the output_dir as a separate argument for future work. Ideally, the output_dir should be the "sample" directory, i.e. the authentication scores should be aggregated across taxa so that we have just one authentication_scores.txt files per sample